libethereum/ChainParams.cpp

```diff
@@ -47,47 +83,75 @@ ChainParams::ChainParams()
 	precompiled.insert(make_pair(Address(4), PrecompiledContract(15, 3, PrecompiledRegistrar::executor("identity"))));
 }
 
-ChainParams::ChainParams(std::string const& _json, h256 const& _stateRoot)
+ChainParams::ChainParams(string const& _json, h256 const& _stateRoot)
+{
+	loadConfig(_json, _stateRoot);
+}
+
+ChainParams::ChainParams(ChainParams const& _org)
+{
+	copy(_org);
+}
+
+void ChainParams::copy(ChainParams const& _org)
```

Comment:
Why not put that directly in the copy constructor? You could use member initializer for most of the fields.